tests/contrib/operators/test_dataproc_operator.py

```diff
@@ -492,6 +522,27 @@ def test_update_cluster(self):
                 })
             hook.wait.assert_called_once_with(self.operation)
 
+    def test_render_template(self):
```

Comment:
This test is a bit silly, maybe we can check if we can actually template a field.

Reply:
They are actually checking to template :D

`DataprocClusterScaleOperator` is passed  `GCP_PROJECT_TEMPLATED`, `CLUSTER_NAME_TEMPLATED` etc which are defined at the top of the file containing Jinja templated strings.

And while asserting we check `GCP_PROJECT` & ` `CLUSTER_NAME` containing rendered fields